sdk/synapse/Microsoft.Azure.Management.Synapse/src/Generated/Models/OperationResource.cs

```diff
@@ -86,7 +86,7 @@ public OperationResource(string id = default(string), string name = default(stri
         /// Gets or sets errors from the operation
         /// </summary>
         [JsonProperty(PropertyName = "error")]
-        public ErrorDetail Error { get; set; }
+        public ErrorResponse Error { get; set; }
```

Comment:
this will result in a breaking change, please revert or have the breaking change team add the approval label to the spec pr